composer.json

```diff
@@ -26,6 +26,7 @@
         "php-amqplib/php-amqplib": "~2.4",
         "swiftmailer/swiftmailer": "~5.3",
         "php-console/php-console": "^3.1.3",
+        "symfony/var-dumper": ">=2.7",
```

Comment:
3.x & 4.x is guaranteed by Symfony BC policy regarding major versions and by the fact that a simple `instanceof` is used.